### PR TITLE
fixed the parameter transfer problem for criterion

### PR DESCRIPTION
**Pull request by yangyyt**
In wekws/kws/utils/executor.py, line 48:
loss, acc = criterion(loss_type, logits, target, feats_lengths)
change to:
loss, acc = criterion(loss_type, logits, target, feats_lengths, min_duration)

### CODE REVIEW

**Review by mlxu995 — APPROVED** [review]
Thank you for your volunteer work.
